src/Microsoft.AspNetCore.Routing/RouteEndpoint.cs

```diff
@@ -26,13 +26,13 @@ public sealed class MatcherEndpoint : Endpoint
         /// The <see cref="EndpointMetadataCollection"/> or metadata associated with the endpoint.
         /// </param>
         /// <param name="displayName">The informational display name of the endpoint.</param>
-        public MatcherEndpoint(
+        public RouteEndpoint(
             RequestDelegate invoker,
```

Comment:
Is there any desire to change the order of RouteEndpoint ctor parameters now that invoker is used on the base ctor?

Reply:
I don't know. Earlier you mentioned you wanted the ctor order to match the base `Endpoint`.
Currently request delegate is first and the other two parameters are last.